src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -215,17 +237,37 @@ private static unsafe nuint GetIndexOfFirstNonAsciiByte_Default(byte* pBuffer, n
             goto Finish;
         }
 
-        private static unsafe nuint GetIndexOfFirstNonAsciiByte_Sse2(byte* pBuffer, nuint bufferLength)
+        private static bool ContainsNonAsciiByte(uint currentSseMaskOrAdvSimdIndex)
```

Comment:
>ContainsNonAsciiByte [](start = 28, length = 20)

I would just separate the 2 methods `ContainsNonAsciiByte_Sse2` and `ContainsNonAsciiByte_AdvSimd` because they both take different type of values.

Reply:
Done.